bazel/external/quiche.BUILD

```diff
@@ -3503,7 +3506,10 @@ envoy_cc_test(
         "quiche/http2/test_tools/http2_random_test.cc",
     ],
     repository = "@envoy",
-    tags = ["nofips"],
+    tags = [
+        "nofips",
+        "test_included",
```

Comment:
Instead of tags, can you add a `test_suite` in somewhere in //test?
https://docs.bazel.build/versions/master/be/general.html#test_suite

Reply:
done